src/BehaviorRelay.ts

```diff
@@ -11,28 +11,42 @@ import { Subscriber, Subscription } from 'rxjs';
  * @class BehaviorRelay<T>
  */
 export class BehaviorRelay<T> extends Relay<T> {
+  private _hasValue = false;
 
-  constructor(private _value: T) {
+  constructor(private _value?: T) {
     super();
+    if (typeof _value !== 'undefined') {
+      this._hasValue = true;
+    }
   }
 
-  get value(): T {
+  get value(): T | void {
     return this.getValue();
   }
 
   _subscribe(subscriber: Subscriber<T>): Subscription {
     const subscription = super._subscribe(subscriber);
-    if (subscription && !subscription.closed) {
+    if (subscription && !subscription.closed && this.hasValue()) {
       subscriber.next(this._value);
     }
     return subscription;
   }
 
-  getValue(): T {
+  getValue(): T | void {
     return this._value;
   }
 
+  hasValue(): boolean {
```

Comment:
I don't want to add this API, for two reasons.
1. It diverge the API from `BehaviorSubject`, which I want to avoid unless we are adding significant value.
2. The value added doesn't warrant the added code complexity IMO.  If people want this functionality, they can call the value getter and use their own logic to determine if that is a valid value.

Reply:
Alright, fair enough 😃 . I'll create another PR with README updates that correctly reflects the current API